repositories/AnswerRepository.js

```diff
@@ -4,6 +4,11 @@ const mapFields = require("../utils/mapFields");
 const mapping = { QuestionPageId: "questionPageId" };
 const fromDb = mapFields(mapping);
 const toDb = mapFields(invert(mapping));
+const db = require("../db");
+const {
+  createOtherAnswer,
+  deleteOtherAnswer
+} = require("./strategies/multipleChoiceOtherAnswerStrategy");
 
 module.exports.findAll = function findAll(
```

Comment:
Do we need to exclude "Other" answers here? when selecting regular answers, you want to filter `where({ otherAnswerId : null })`?

Reply:
Yeah good point.

Reply:
I implemented this change, then reverted it, since it broke the resolver for `otherAnswer` property on `MultipleChoiceAnswer` (i.e. prevents you from doing what I mentioned above!).

Filtering out "other" answers should be a concern of resolvers that need to do so.